docs/SystemEdls.md

```diff
@@ -136,6 +136,7 @@ oe_syscall_getnameinfo_ocall | N/A | Used by internal APIs to resolve `addrinfo`
 Ocall | Dependent syscall | Comments |
 :---|:---:|:---|
 oe_syscall_nanosleep_ocall | nanosleep | - |
+oe_syscall_clock_nanosleep_ocall | clock_nanosleep | In Windows, only flag = 0 is supported. |
```

Comment:
```suggestion
oe_syscall_clock_nanosleep_ocall | clock_nanosleep | In Windows, only flag = 0 and clockid = CLOCK_THREAD_CPUTIME_ID are supported. |
```

Reply:
CLOCK_THREAD_CPUTIME_ID is not supported in Linux as well. As it is an expected behavior, did not mention it here.